torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -1080,10 +1080,9 @@ def test_backward_node_failure(self):
                         wait_until_node_failure(rank)
 
                 # Shutdown sequence is not very well defined and as a result
-                # we might see either of the exception messages below.
-                with self.assertRaisesRegex(RuntimeError,
-                                            "(Request aborted during client shutdown)|"
-                                            "(worker.: Error in reponse from worker.: server shutting down)"):
+                # we might see any error given by get_shutdown_error_regex()
+                error_regex = get_shutdown_error_regex()
+                with self.assertRaisesRegex(RuntimeError, error_regex):
```

Comment:
nit: just use `get_shutdown_error_regex` inline instead of having a separate `error_regex` variable.